dind-wrapper.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+echo "==> Launching the Docker daemon..."
+if docker info > /dev/null 2>&1; then
+    echo "=== Docker already running";
+else
+    /usr/local/bin/dind dockerd $DIND_DOCKER_ARGS > /var/log/docker.log 2>&1 &
+    for i in `seq 30`; do
```

Comment:
lets make this a higher value (300 seconds may be?), i don't see any downside in making this a higher value.